config/authentication.py

```diff
@@ -208,6 +209,56 @@ def get_user(self, validated_token):
         return user
 
 
+class HCXAuthentication(JWTAuthentication):
```

Comment:
add a simple schema definition class for swagger [ref](https://drf-spectacular.readthedocs.io/en/latest/customization.html?highlight=OpenApiAuthenticationExtension#specify-authentication-with-openapiauthenticationextension)

similar to
https://github.com/coronasafe/care/blob/0143084df7e994e2a45624a8ce684880782f1bf2/config/authentication.py#L272-L289